Orange/data/io.py

```diff
@@ -591,14 +524,14 @@ def _vartype(var):
             elif var.is_discrete:
                 return Flags.join(var.values) if var.ordered else var.TYPE_HEADERS[0]
             raise NotImplementedError
-        return ['continuous'] * data.has_weights() + \
+        return ['continuous'] + \
```

Comment:
So when the data is saved, the new column is always prepended even when weights are irrelevant? I don't like this. The weights should be stripped when irrelevant. Perhaps a `has_weights()` method on the data frame that checks `W is not None and len(W) and not (W == W[0]).all()`?